client/sagas/user.js

```diff
@@ -43,10 +47,44 @@ export function* userLoginSaga() {
 
   const { accessToken, idToken } = login
 
+  yield call(firebaseLoginAndRedirect, 'Google', idToken, accessToken)
```

Comment:
I think you don't need to use `call` here, you could just do
```suggestion
  yield* firebaseLoginAndRedirect('Google', idToken, accessToken)
```
The same applies to the facebook saga